torch/_inductor/utils.py

```diff
@@ -732,10 +741,10 @@ def __len__(self):
 
 
 @functools.lru_cache(None)
-def is_big_gpu(index):
```

Comment:
I'll try to move these changes into a separate PR after some investigation.

I have a feeling we can get rid of this check entirely.

Reply:
There is a PR and ongoing investigation now: https://github.com/pytorch/pytorch/pull/109492